packages/driver/src/cy/multidomain/index.ts

```diff
@@ -127,9 +148,18 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
 
           // once the secondary domain page loads, send along the
           // user-specified callback to run in that domain
-          communicator.toSpecBridge('run:domain:fn', {
-            fn: fn.toString(),
-          })
+          try {
+            communicator.toSpecBridge('run:domain:fn', {
```

Comment:
Does this have the effect we think it does here? I would think this only errors if there are issues with `postMessage` under the hood to the secondary domain. Do we need to listen to the `ran:domain:fn` event and then assert if any errors were returned like [here](https://github.com/cypress-io/cypress/pull/19465/files#diff-2e175972501a2537ec1a0269982d821197d7e27506906afc881515f63f050679R130)?

I also noticed it's `ran:domain:fn` above over `run:domain:fn` in this branch, which might be out of sync with the changes I am thinking of. Too many branches! 😂

Reply:
Yes, absolutely. I didn't implement that here since I knew you had already done so in the done callback branch. 

I think we renamed both to `run:domain:fn` when we were trying to promisify things. Since that didn't work out, I think it makes more sense to have the reply be `ran:domain:fn`.